examples/bpk-component-graphic-promotion/examples.js

```diff
@@ -22,8 +22,12 @@ import BpkGraphicPromo, {
   TEXT_ALIGN,
 } from '../../packages/bpk-component-graphic-promotion';
 
-const image =
-  'https://content.skyscnr.com/m/31ebf33b07194794/original/Hiker-looking-out-over-mountain.jpg?crop=1048px:400px&quality=90';
+const backgroundImageDesktop =
+  'https://content.skyscnr.com/m/31ebf33b07194794/original/Hiker-looking-out-over-mountain.jpg?crop=2096px:800px&quality=90';
+const backgroundImageTablet =
+  'https://content.skyscnr.com/m/31ebf33b07194794/original/Hiker-looking-out-over-mountain.jpg?crop=820px:1180px&quality=90';
+const backgroundImageMobile =
+  'https://content.skyscnr.com/m/31ebf33b07194794/original/Hiker-looking-out-over-mountain.jpg?crop=390:844px&quality=90';
```

Comment:
This is something I recently spoke to design about and using the correct ratios for the images, in the end we found `1048px:400px` is the one we should use which gives an aspect ratio that works across the different breakpoints - the guidance for this should be available in the Backpack figma libraries

Reply:
Resolving this since we've taken the approach of using 3 different images